src/Oscar.jl

```diff
@@ -39,21 +39,12 @@ if Sys.iswindows()
 end
 
 function _print_banner()
-  println(" -----    -----    -----      -      -----   ")
-  println("|     |  |     |  |     |    | |    |     |  ")
-  println("|     |  |        |         |   |   |     |  ")
-  println("|     |   -----   |        |     |  |-----   ")
-  println("|     |        |  |        |-----|  |   |    ")
-  println("|     |  |     |  |     |  |     |  |    |   ")
-  println(" -----    -----    -----   -     -  -     -  ")
-  println()
-  println("...combining (and extending) ANTIC, GAP, Polymake and Singular")
-  print("Version")
-  printstyled(" $VERSION_NUMBER ", color = :green)
-  print("... \n ... which comes with absolutely no warranty whatsoever")
-  println()
-  println("Type: '?Oscar' for more information")
-  println("(c) 2019-2024 by The OSCAR Development Team")
+  println(
+    raw"""  ___  ____   ____    _    _.__    |  Combining ANTIC, GAP, Polymake, Singular
+           / | \/ ___| / ___|  / \  | |_ \   |  Type "?Oscar" for more information
+          | | | \_|_ \| |     / . \ | |_) |  |  Manual: https://docs.oscar-system.org
+          | |_| |___) | |___ / /_\ \| |_ <   |
+           \_|_/\____/ \____/_/ _ \_\_| \_\  |  Version """ * $VERSION_NUMBER)
```

Comment:
You have to enclose version number in string

```suggestion
           \_|_/\____/ \____/_/ _ \_\_| \_\  | Version """ * "$VERSION_NUMBER")
```